### PR TITLE
Add weapon sound

### PR DESCRIPTION
**Pull request by ligamesh123**

### CODE REVIEW

**Comment by xix-xeaon** [comment]
This PR will have to wait until the weapon has been merged. It might make more sense to create a new PR then which also plays the sound when it fires.

File: project.godot
Comment (ExclusiveKnight) [inline project.godot:48]: Why add pause??? It's not relevant to this PR. Instead of using .wav for the sound files, use .ogg format instead as it offers compression (see /Sounds folder in the repository for reference). The .wav format offers higher sound quality at the expense of higher file size.

**Review by ExclusiveKnight — CHANGES_REQUESTED** [review]

**Comment by ligamesh123** [comment]
> This PR will have to wait until the weapon has been merged. It might make more sense to create a new PR then which also plays the sound when it fires.

I have added weapon sound to raygun_scene

**Pull request merged**
